Makefile

```diff
@@ -76,7 +76,7 @@ serve:
 	node server.js
 
 test:
-	./node_modules/.bin/jasmine-node test/katex-spec.js
+	./node_modules/.bin/jasmine-node test/*-spec.js
```

Comment:
That adds a dependency on findutils. Should be no problem on \*nix, but I'm less sure about what one can reasonably expect on Windows. And passing `test/*/*-spec.js` isn't a problem either, once we have an extra level. Even if we had to choose the recursive approach, I'd prefer a single command instead of three: `find test/ -name \*-spec.js -exec ./node_modules/.bin/jasmine-node '{}' +`. But let's worry about these if we ever need to.

What I found confusing during my tests was that a spec file with broken syntax apparently prevents jasmin from processing that file, but doesn't result in a non-zero exit code. So a commit might accidentially break a whole test file and a glance at the travis status wouldn't tell us about this. Will investigate and file a bug there if I can confirm this.